Makefile

```diff
@@ -28,9 +28,14 @@ build:
 	rm -rf ./dist
 	hugo -s site --verbose --destination ../dist/html --printUnusedTemplates --printPathWarnings --gc
 
+
+.PHONY: test
+test:
+	htmlproofer ./dist/html --ignore_empty_alt
```

Comment:
This command fails. How do I get the dependencies I need to run the command?

Reply:
Maybe we should make use of https://html-validate.org/usage/cli.html as it's a js tool and therefore dev already installed the tools needed to run it.